Pdf/Model/Order.php

```diff
@@ -0,0 +1,128 @@
+     * @param \Magento\Store\Model\StoreManagerInterface $storeManager
+     * @param \Magento\Framework\Locale\ResolverInterface $localeResolver
+     * @param array $data
+     */
+    public function __construct(
+        \Magento\Payment\Helper\Data $paymentData,
+        \Magento\Framework\Stdlib\StringUtils $string,
+        \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig,
+        \Magento\Framework\Filesystem $filesystem,
+        \Magento\Sales\Model\Order\Pdf\Config $pdfConfig,
+        \Magento\Sales\Model\Order\Pdf\Total\Factory $pdfTotalFactory,
+        \Magento\Sales\Model\Order\Pdf\ItemsFactory $pdfItemsFactory,
+        \Magento\Framework\Stdlib\DateTime\TimezoneInterface $localeDate,
+        \Magento\Framework\Translate\Inline\StateInterface $inlineTranslation,
+        \Magento\Sales\Model\Order\Address\Renderer $addressRenderer,
+        \Magento\Store\Model\StoreManagerInterface $storeManager,
+        \Magento\Framework\Locale\ResolverInterface $localeResolver,
+        Data $_helper,
+        array $data = []
+    )
+    {
+        $this->_rootDirectory = $filesystem->getDirectoryRead(DirectoryList::ROOT);
+        $this->_helper = $_helper;
+        parent::__construct(
+            $paymentData,
+            $string,
+            $scopeConfig,
+            $filesystem,
+            $pdfConfig,
+            $pdfTotalFactory,
+            $pdfItemsFactory,
+            $localeDate,
+            $inlineTranslation,
+            $addressRenderer,
+            $storeManager,
+            $localeResolver,
+            $data
+        );
+    }
+
+    /**
+     * @return Data
+     */
+    public function getHelper()
+    {
+        return $this->_helper;
+    }
+
+    /**
+     * @param $object
+     * @param int $size
+     * @return mixed
+     */
+    protected function _setFontRegular($object, $size = 7)
+    {
+        if ($this->getHelper()->getJapaneseFontIsActive()) {
+            $fontpath = $this->getHelper()->getJapaneseFont();
+            $font = \Zend_Pdf_Font::fontWithPath($fontpath);
+            $object->setFont($font, $size);
+            return $font;
+        }
+        return parent::_setFontRegular($object, $size);
+    }
+
+    /**
+     * @param $object
+     * @param int $size
+     * @return mixed
+     */
+    protected function _setFontBold($object, $size = 7)
+    {
+        if ($this->getHelper()->getJapaneseFontIsActive()) {
+            $fontpath = $this->getHelper()->getJapaneseFont();
+            $font = \Zend_Pdf_Font::fontWithPath($fontpath);
+            $object->setFont($font, $size);
+            return $font;
+        }
+        return parent::_setFontBold($object, $size);
+    }
+
+    /**
+     * @param $object
+     * @param int $size
+     * @return mixed
+     */
+    protected function _setFontItalic($object, $size = 7)
+    {
+        if ($this->getHelper()->getJapaneseFontIsActive()) {
+            $fontpath = $this->getHelper()->getJapaneseFont();
```

Comment:
This part (following 3 lines) seems to be shared across multiple classes and methods. It makes sense to extract it.